output/schema/schema.json

```diff
@@ -72046,7 +72046,7 @@
         "name": "CombinedFieldsOperator",
         "namespace": "_types.query_dsl"
       },
-      "specLocation": "_types/query_dsl/abstractions.ts#L473-L476"
+      "specLocation": "_types/query_dsl/abstractions.ts#L481-L484"
```

Comment:
**Non-blocking question**
How did we regenerate the schema.json file?

Reply:
The instructions on how to do this are in the [README](https://github.com/elastic/elasticsearch-specification/blob/main/README.md).